lib/elixir/lib/port.ex

```diff
@@ -3,56 +3,79 @@ defmodule Port do
   Functions related to Erlang ports.
   """
 
+  @type name :: {:spawn, string | binary} |
```

Comment:
`string` should be `char_list` 
From the `Kernel.Typespec` Module: 
> Elixir discourages the use of type string as it might be confused with binaries which are referred to as “strings” in Elixir (as opposed to character lists). In order to use the type that is called string in Erlang, one has to use the char_list type which is a synonym for string. If you use string, you’ll get a warning from the compiler.